docs/docs/pods.md

```diff
@@ -0,0 +1,583 @@
+
+The following JSON specifies a Docker image for the pod:
+
+```json
+{  
+   "image":{  
+      "id":"mesosphere/marathon:latest",
+      "kind":"DOCKER",
+      "forcePull":false
+   }
+}
+```
+
+### Network
+
+# Create and Manage Pods
+
+Use the `/v2/pods/` endpoint to create and manage your pods. [See the full API spec](http://mesosphere.github.io/marathon/docs/generated/api.html#v2_pods).
+
+## Create
+
+```json
+ $ http POST <ip>:<port>/v2/pods mypod.json
+```
+
+Sample response:
+
+```json
+HTTP/1.1 201 Created
+Cache-Control: no-cache, no-store, must-revalidate
+Content-Type: application/json; qs=2
+Date: Tue, 18 Oct 2016 21:07:49 GMT
+Expires: 0
+Location: http://localhost:8080/v2/pods/simplepod2
+Marathon-Deployment-Id: 8b591de8-6f2d-4036-870e-7571912d3522
+Pragma: no-cache
+Server: Jetty(9.3.z-SNAPSHOT)
+Transfer-Encoding: chunked
+X-Marathon-Leader: http://71c812cd6810:8080
+
+{
+    "containers": [
+        {
+            "artifacts": [],
+            "endpoints": [],
+            "environment": {},
+            "exec": {
+                "command": {
+                    "shell": "sleep 1000"
+                }
+            },
+            "healthCheck": null,
+            "image": null,
+            "labels": {},
+            "lifecycle": null,
+            "name": "sleep1",
+            "resources": {
+                "cpus": 0.1,
+                "disk": 0,
+                "gpus": 0,
+                "mem": 32
+            },
+            "user": null,
+            "volumeMounts": []
+        }
+    ],
+    "environment": {},
+    "id": "/simplepod2",
+    "labels": {},
+    "networks": [
+        {
+            "labels": {},
+            "mode": "host",
+            "name": null
+        }
+    ],
+    "scaling": {
+        "instances": 2,
+        "kind": "fixed",
+        "maxInstances": null
+    },
+    "scheduling": {
+        "backoff": {
+            "backoff": 1,
+            "backoffFactor": 1.15,
+            "maxLaunchDelay": 3600
+        },
+        "placement": {
+            "acceptedResourceRoles": [],
+            "constraints": []
+        },
+        "upgrade": {
+            "maximumOverCapacity": 1,
+            "minimumHealthCapacity": 1
+        }
+    },
+    "secrets": {},
+    "user": null,
+    "version": "2016-10-18T21:07:49.94Z",
+    "volumes": []
+}
+```
+
+## Status
+
+Get the status of all pods:
+
+```bash
+http GET <ip>:<port>/v2/pods/::status
+```
+
+Get the status of a single pod:
+
+```bash
+http GET <ip>:<port>/v2/pods/<pod-id>::status
+```
+
+## Delete
+
+```bash
+http DELETE <ip>:<port>/v2/pods/<pod-id>
+```
+
+# Example Pod Definitions
+
+## A Pod with Multiple Containers
+	
+The following pod definition specifies a pod with 3 containers.
+
+```json
+{
+  "id": "/pod-with-multiple-containers",
+  "labels": {
+    "values": {}
+  },
+  "version": "2016-09-22T09:18:05.928Z",
```

Comment:
this json looks like a response you got... you would NOT specify a version in your json definition of pods.